.github/workflow_scripts/pytest_check.sh

```diff
@@ -1,9 +1,20 @@
-# Move to parent directory
+#!/bin/env bash
+# Move to repository root
+set -ex
+
 cd ../../
 
-set -ex
 
-FORCE_CUDA=1 python3 -m pip install -e '.[test]'  --no-build-isolation
+GS_HOME=$(pwd)
+# Add SageMaker launch scripts to make the scripts testable
+export PYTHONPATH="${PYTHONPATH}:${GS_HOME}/sagemaker/launch/"
+
 python3 -m pip install pytest
+FORCE_CUDA=1 python3 -m pip install -e '.[test]'  --no-build-isolation
+
+# Run SageMaker tests
+python3 -m pytest -x ./tests/sagemaker-tests -s
```

Comment:
I was thinking to have them before because they take trivial time so for someone only touching SageMaker code they can fail early and not wait an hour for the main tests to run?

Reply:
But most of the time, we do not change the SageMaker code.
Does unit test take an hour?